src/index.js

```diff
@@ -1,2 +1,45 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+  Object.entries(props || {}).forEach(([key, value]) => { element[key.toLowerCase()] = value; });
+  children.flat().forEach((child) => {
```

Comment:
선언과 액션들 사이에는 한줄씩 띄어주시면 가독성이 훨씬 좋아진답니다 :)
```suggestion
function createElement(tagName, props, ...children) {
  const element = document.createElement(tagName);
  
  Object.entries(props || {}).forEach(([key, value]) => { 
    element[key.toLowerCase()] = value; 
  });
  
  children.flat().forEach((child) => {
```